TEMPLATES/mega-linter.yml

```diff
@@ -108,61 +264,87 @@ jobs:
             megalinter-reports
             mega-linter.log
 
-      # Set APPLY_FIXES_IF var for use in future steps
```

Comment:
Freshen this PR from main to avoid undoing https://github.com/oxsecurity/megalinter/pull/2957